src/test/java/seedu/address/nusmods/NusmodsDataManagerTest.java

```diff
@@ -23,6 +23,10 @@
     private NusmodsDataManager manager = new NusmodsDataManager(new ReadFromFileDataFetcherStub(),
                                                                 VALID_MAP_TEST_FILE_PATH);
 
+    public NusmodsDataManager getStubManager() {
```

Comment:
I think it's better to make this a static method since it is a getter for a static stub class; otherwise we'll have to do `new NusmodsDataManagerTest` everytime we want to get the stub

Reply:
Yeap sounds good!